R/compat-name-repair.R

```diff
@@ -81,28 +82,27 @@ make_syntactic <- function(name) {
   ##   * turned its '.' suffixes to '.' prefixes
 
   regex <- paste0(
-    "^(?<leading_dots>[.]{0,2})",
+    "^(?<leading_dots>[.]{0,3})",
     "(?<numbers>[0-9]*)",
```

Comment:
Does anything in `make_syntactic()` need to change, actually?

Reply:
I've thought and written about this more and I'm increasingly convinced nothing should change in `make_syntactic()`.